src/room/GroupCallLoader.tsx

```diff
@@ -38,6 +41,9 @@ export function GroupCallLoader({
   const { t } = useTranslation();
   const groupCallState = useLoadGroupCall(client, roomIdOrAlias, viaServers);
 
+  const history = useHistory();
+  const onHomeClick = useCallback(() => history.push("/"), [history]);
```

Comment:
I think if you include an `event.preventDefault()` in this handler, that will keep the page from refreshing unnecessarily

Reply:
Ah, of course - thanks.